src/index.js

```diff
@@ -11,7 +11,11 @@ const messages = utils.ruleMessages(ruleName, {
   expected,
 })
 const reSkipProp = /^(?:@|\$|--).+$/
-const reVar = /^-?(?:@.+|\$.+|var\(--.+\))$/
+// Sass namespaces and CSS <ident-token>
+// @see: https://github.com/sass/sass/blob/master/accepted/module-system.md#member-references
+// @see:  https://drafts.csswg.org/css-syntax-3/#ident-token-diagram
+// eslint-disable-next-line no-control-regex
+const reVar = /^-?(?:@.+|(?:(?:[a-zA-Z_-]|[^\x00-\x7F])+(?:[a-zA-Z0-9_-]|[^\x00-\x7F])*\.)?\$.+|var\(--.+\))$/
```

Comment:
@thibaudcolas 
Alright, I I added an `<ident-token>` regex for the sass namespace, includeing Non-ASCII range.
Thank you!